### PR TITLE
Project 3 add and remove

### PR DESCRIPTION
**Pull request by SamanAtashi**
# Project 3: add and remove

>  Made with:
- HTML
- CSS
- JS
- WebPack

> Added functionalities:
- Local storage update 
- clear all completed btn

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ import './style.css';
 // eslint-disable-next-line import/no-cycle
 import { statusUpdate } from './statusUpdate.js';
 import { dragFunction } from './dragDropSort.js';
+// eslint-disable-next-line import/no-unresolved
+// eslint-disable-next-line import/no-cycle
```
Comment (Dipeshtwis) [inline src/index.js]: [OPTIONAL]
- I can see you have disabled many rules in your project. I highly recommend to not disable many rules instead you can try to make your code good and follow the best practices. The purpose is to make your code better.

File: src/index.js [hunk 2]
```diff
@@ -97,3 +84,57 @@ document.addEventListener('click', (e) => {
...
+    }
+
+    if (parent.classList.value === 'item d-flex red') {
+      parent.insertBefore(createDeleteBtn(), e.target);
+    } else {
+      // eslint-disable-next-line no-lonely-if
+      if (previous.classList.value === 'dynamicDeleteBtn') {
+        previous.remove();
+      }
+    }
```
Comment (Dipeshtwis) [inline src/index.js]: - [x]  Please use `localstorage` to store the data so that it persist after refresh as well.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @SamanAtashi, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Able to add and remove the task.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] I suggest that your pull request be more descriptive describing introduced changes in more details this is a recommended best practice as it makes reviewing easier.
- [x] All changes to the To Do List should be saved in local storage. currently no changes in happening in localstorage.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @SamanAtashi  🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

boomeasy

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**